tools/ms2snoop/MS2snoop.xml

```diff
@@ -143,14 +149,14 @@
         </section>
     </inputs>
     <outputs>
-        <data name="frag_result_txt" format="tsv" label="${tool.name} on ${peaklist_frag_tsv.name}" />
-        <data name="pdf_output" format="zip" label="${tool.name} PDFs for ${peaklist_frag_tsv.name}">
+        <data name="frag_result_txt" format="tsv" label="${tool.name}_on_${peaklist_frag_tsv.name}.tsv" />
+        <data name="pdf_output" format="zip" label="${tool.name}_PDFs_for_${peaklist_frag_tsv.name}.zip">
```

Comment:
are the underscores and the extension needed? Those are not filenames, but Galaxy metadata.

Reply:
You're right, it's superfluous.

Reply:
Do you need the `_` in the names?